### ISSUE TITLE
[v1] Nightwatch should be inside test?

### ISSUE THREAD

**Issue by seaneking** [SimplaElements/simpla-text#47]
Just a minor code organisation thing - shouldn't `nightwatch/` be inside `test/`?

**Comment by bedeoverend** [comment]
Yeah it should be, I was just struggling to get the build step working with it properly. It was taking me too much time so decided to fix it down the track
